### ISSUE TITLE
The compare drawer header not visible when the drawer is closed (Collapsed)

### ISSUE THREAD

**Issue by hallihocine** [department-of-veterans-affairs/va.gov-team#34414]
## What happened?
The compare drawer header is changing the background color when the drawer is collapsed, which making it less visible. This will lead to color contrast accessibility failure. See below:

https://user-images.githubusercontent.com/86266024/146967362-dc31f4da-12fa-4b94-8795-f29d2f7c0a35.mov

## Steps to Reproduce
- Navigate to: https://staging.va.gov/gi-bill-comparison-tool-sandbox/?search=name&name=CHICAGO%20STATE%20UNIVERSITY
- Expand (Open) and collapse (Close) the compare institution drawer

## Desired behavior
- The header of the compare institution drawer should be visible when the drawer is collapsed (Closed)

## Acceptance Criteria

- [ ] The header of the compare institution drawer should be visible when the drawer is collapsed (Closed)

**Comment by hallihocine** [comment]
Ticket has been validated in staging: Background color of the compare drawer is set to blue and it's not changing when opening or closing the drawer. 

https://user-images.githubusercontent.com/86266024/149010881-2ec09d6b-0b9c-45c1-b645-8f86f81e3ede.mov

**Comment by hallihocine** [comment]
- Test run in Test Rail:
https://dsvavsp.testrail.io/index.php?/runs/view/2820&group_by=cases:section_id&group_order=asc&group_id=3319
- Test Script/Steps:
[The compare drawer header is less visible when the drawer is closed .xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/7849525/The.compare.drawer.header.is.less.visible.when.the.drawer.is.closed.xlsx)

**Comment by mrblanco1** [comment]
Conducted UAT on 1/18/2022. UAT passed. Closing.
